cmd/gitserver/server/server.go

```diff
@@ -1256,7 +1254,8 @@ func (s *Server) cloneRepo(ctx context.Context, repo api.RepoName, opts *cloneOp
 	}
 
 	if err := syncer.IsCloneable(ctx, remoteURL); err != nil {
-		return "", errors.Errorf("error cloning repo: repo %s not cloneable: %s", repo, redactor.redact(err.Error()))
+		redactedErr := newURLRedactor(remoteURL).redact(err.Error())
```

Comment:
```suggestion
		redactErr := newURLRedactor(remoteURL).redact(err.Error())
```

nit: 'cause the method is named `redact` 😁

Reply:
My thought process was that `redact` applies some transformation on the error message and thus the state is now `redacted` - similar to this: https://github.com/sourcegraph/sourcegraph/blob/14a377fb15470d9a778a629d7c50c55aad66caae/cmd/gitserver/server/server.go

Reply:
Yes you're right, I misunderstood it.